lib/WorkDoneProgress/MessageProgressNotifier.php

```diff
@@ -42,7 +42,17 @@ public function begin(
         ?int $percentage = null,
         ?bool $cancellable = null
     ): void {
-        $this->api->info($message);
+        $progress = [
+            $title
+        ];
```

Comment:
I'm not sure we should include the title.
I think it's ment to have a title in a notification window, when showing a message it might get redundant.
With the example of the indexing we will get something like `Indexing: Indexing X files`

Reply:
I will try to improve it on this branch and push a proposition, you'll decide then if you keep it or not :)

Reply:
Silly me, I forgot I can't work on this branch directly :smile: 
https://github.com/camilledejoye/language-server/commit/40f222cd9dbddfdc4e25e8f2d9f6aeac528e75fc